theano/tensor/nnet/nnet.py

```diff
@@ -2448,6 +2448,29 @@ def elu(x, alpha=1):
     return tensor.switch(x > 0, x, alpha * tensor.expm1(x))
 
 
+def selu(x, alpha=1):
+    """Compute the element-wise Scaled Exponential Linear unit.
+
+    Parameters
+    ----------
+    x : symbolic tensor
+        Tensor to compute the activation function for.
+
+    Returns
+    -------
+    symbolic tensor
+        Element-wise scaled exponential linear activation function applied to `x`.
+
+    References
+    ----------
+    .. [1] Klambauer G, Unterthiner T, Mayr A, Hochreiter S.
+        "Self-Normalizing Neural Networks" <https://arxiv.org/abs/1706.02515>
+    """
+    alpha = 1.6732632423543772848170429916717
```

Comment:
Here you override the alpha from the function signature. If you want this value to be the default, set it in the fct signature.

A quick look at the other framework interface make me think we should just remove the alpha parameter from the fct signature.